src/main/java/com/ninecookies/wiremock/extensions/api/Callback.java

```diff
@@ -54,6 +58,22 @@ public static Callback ofQueueMessage(int delay, String queue, Object message) {
         return result;
     }
 
+    /**
+     * Create a new instance for an SNS message {@link Callback} definition.
+     *
+     * @param delay the period of time in milliseconds to wait before the callback data published.
+     * @param topic the destination topic name to send the message to.
+     * @param message an arbitrary JSON object representing the message to send.
+     * @return a new {@link Callback} instance ready to use.
+     */
+    public static Callback ofTopicMessage(int delay, String topic, Object message) {
```

Comment:
```suggestion
    public static Callback ofSNSMessage(int delay, String topic, Object message) {
```
?

Reply:
This is public API. We have `ofQueueMessage()`, thus here the `ofTopicMessage()` instead of concreter SNS / SQS names for the methods.
Could be e.g. Redis support in future as well enhanced with a type property to distinguish between AWS and Redis messaging. Therefore the more generic names... ;)
WDYT?

Reply:
alrighty then :)